homeassistant/components/sensor/dyson.py

```diff
@@ -14,12 +14,20 @@
 DEPENDENCIES = ['dyson']
 
 SENSOR_UNITS = {
-    'air_quality': 'level',
-    'dust': 'level',
+    'air_quality': '',
```

Comment:
I assume `None` is preferred instead of an empty string.

Reply:
Well spotted, I see other sensors having `None` indeed as unit. Fixed by the next commit.